application/common/models/EmailLog.php

```diff
@@ -31,6 +31,7 @@ class EmailLog extends EmailLogBase
     const MESSAGE_TYPE_PASSWORD_EXPIRING = 'password-expiring';
     const MESSAGE_TYPE_PASSWORD_EXPIRED = 'password-expired';
     const MESSAGE_TYPE_PASSWORD_PWNED = "password-pwned";
+    const MESSAGE_TYPE_UNUSED_USERS = 'unused-users';
```

Comment:
I suggest to not to define this here since it is not a logged email type. Maybe even better would be to proceed with this implementation by adding this type to the database enum and find a way to actually log this activity. (Log with a null user ID for example.)

Reply:
Moved out of EmailLog for now.